product-service/serverless.yml

```diff
@@ -0,0 +1,146 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: product-service
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: "3"
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: eu-west-1
+
+custom:
+  webpack:
+    includeModules: false
+
+package:
+  individually: true
+
+plugins:
+  - serverless-auto-swagger
```

Comment:
You can configure this plugin to generate swagger docs with response types and with auto deploy (Swagger docs will be available in the same API Gateway. I see that your docs are hosted in the separate API Gateway). Check the plugin docs.
If you want an example you can check my PR (https://github.com/SergeyShmaro/rs-shop-aws-backend/pull/1).